BraveRewardsUI/Common/LinkLabel.swift

```diff
@@ -10,10 +10,9 @@ final class LinkLabel: UITextView {
   /// Called when a link is tapped
   var onLinkedTapped: ((URL) -> Void)?
   
-  override var text: String? {
-    didSet {
-      updateText()
-    }
+  func setURLs(text: String, urlInfo: [String: String]) {
```

Comment:
If we're gonna name this `setURLs`, can't we just only send in the URL mapping and set the text as normal?